stories/core/Footer.stories.tsx

```diff
@@ -46,6 +46,39 @@ Custom.args = {
   ],
 } as FooterProps;
 
+export const OnlyCustomElements: Story<FooterProps> = ({
+  customElements,
+  showOnlyCustomElements,
+  ...rest
+}: FooterProps) => {
+  return (
+    <Footer
+      customElements={customElements}
+      showOnlyCustomElements={showOnlyCustomElements}
```

Comment:
Just enough having `showOnlyCustomElements`, for the same readability

Reply:
Fixed